digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -268,7 +268,12 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
               logger.info("Added links recognized to be a roundabout - using roundabout addressing scheme")
               val ordered = newProjectLinks.values.toSeq.partition(_.linkId == firstLinkId)
               val created = TrackSectionOrder.mValueRoundabout(ordered._1 ++ ordered._2)
-              created
+              val endingM = created.map(_.endAddrMValue).max
```

Comment:
i think maxBy can be used here

Reply:
Could be, probably, but won't be any cleaner.